src/torchmetrics/__init__.py

```diff
@@ -21,26 +21,19 @@
     SignalNoiseRatio,
 )
 from torchmetrics.classification import (  # noqa: E402
-    AUC,
     AUROC,
     ROC,
     Accuracy,
     AveragePrecision,
-    BinnedAveragePrecision,
-    BinnedPrecisionRecallCurve,
-    BinnedRecallAtFixedPrecision,
     CalibrationError,
     CohenKappa,
     ConfusionMatrix,
-    CoverageError,
     Dice,
     F1Score,
     FBetaScore,
     HammingDistance,
     HingeLoss,
     JaccardIndex,
-    LabelRankingAveragePrecision,
-    LabelRankingLoss,
```

Comment:
do we have replacements for these?

Reply:
yes, they have just been renamed to streamline:
https://github.com/Lightning-AI/metrics/blob/fbfe974e44b157819bbe5421494e616bf63c6e6f/src/torchmetrics/classification/__init__.py#L99-L100

Reply:
good, let's do https://github.com/Lightning-AI/metrics/pull/1251#discussion_r992214449